File: client/src/models/TodoListItem.js
export class TodoListItem {
  constructor(data) {
    this.id = data.id
    this.createdAt = data.createdAt
    this.updatedAt = data.updatedAt
    this.completed = data.completed
    this.description = data.description
    this.creatorId = data.creatorId
    this.todoListId = data.todoListId
  }
}

File: server/Models/TodoListItem.cs
namespace inspireVue.Models;

public class TodoListItem
{
  public int Id { get; set; }
  public DateTime CreatedAt { get; set; }

  public DateTime UpdatedAt { get; set; }
  public bool? Completed { get; set; }
  public string Description { get; set; }

  public string CreatorId { get; set; }

  public int TodoListId { get; set; }
}
